packages/shared/src/types/rspack.ts

```diff
@@ -113,11 +104,39 @@ export type BuiltinSwcLoaderOptions = {
     };
     preserveAllComments?: boolean;
   };
-  rspackExperiments?: Pick<
-    BuiltinsOptions,
-    'react' | 'decorator' | 'presetEnv' | 'emotion' | 'relay'
-  > & {
-    import?: BuiltinsOptions['pluginImport'];
+  rspackExperiments?: {
+    relay?:
```

Comment:
Can we import relay / emotion / import options from Rspack?

Reply:
Let's do it after 0.5 release